### PR TITLE
Goodbye Frank. We'll miss you. 💔

### PR DESCRIPTION
**Pull request by mattr-**
Perhaps this post could be longer, but I don't really know what else to say.

### CODE REVIEW

**Review by parkr — APPROVED** [review]
💔 😢

File: docs/_posts/2021-09-14-goodbye-dear-frank.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: 'Goodbye, Dear Frank.'
+date: 2021-09-14 11:28:02 -0500
+author: ashmaroli
+categories: [team, community]
+---
+
+Over the weekend, the Jekyll core team learned of the passing of one of their own: *Frank Taillandier*, popularly known
```
Comment (mattr-) [inline docs/_posts/2021-09-14-goodbye-dear-frank.markdown:8]:
```suggestion
Over the weekend, the Jekyll core team learned of the passing of one of our own: *Frank Taillandier*, popularly known
```

File: docs/_posts/2021-09-14-goodbye-dear-frank.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+novice users yet detailed enough for advanced users seeking additional information.
+
+He also served as the administrator for Jekyll's public [discourse forum](https://talk.jekyllrb.com/) where he not only
+addressed queries from users and provided tips to improve Jekyll workflow, he also shared feedback on Jekyll sites
+created by the community, and used the forum as a platform to gather feedback on unreleased iterations of Jekyll and
+in-house plugins.
+
+Abreast with latest developments in the Web-verse, Frank was always quick to introduce technologies that vastly improved
+maintenance in the Jekyll organization. He was instrumental in setting up deploy previews for patches to Jekyll's
+documentaion site and later wiring GitHub Actions to handle continuous integrations for Jekyll and in-house projects.
```
Comment (mattr-) [inline docs/_posts/2021-09-14-goodbye-dear-frank.markdown:24]:
```suggestion
documentation site and later wiring GitHub Actions to handle continuous integrations for Jekyll and in-house projects.
```

**Comment by ashmaroli** [comment]
@mattr- I rewrote the goodbye note.

**Review by mattr- — COMMENTED** [review]
This is beautiful. I fixed a few typos, but that's it. 

Thanks for taking my poor writing and greatly enhancing it. ❤️

**Comment by ashmaroli** [comment]
@parkr Requesting a re-review. Feel free to squash-merge and publish when ready.
Thank you.

**Comment by mattr-** [comment]
I'm going to go ahead and merge this. 

💔 😢 

@jekyllbot: merge +docs

**Comment by mariealine** [comment]
Hello folks,

I didn't know how to contact you in a more traditional way.
I just read this blog post.
Frank was a close friend.
Reading your words means a lot to me. Thank you a lot for sharing.
Take care

Marie-Aline Millot

**Comment by ashmaroli** [comment]
Hello @mariealine 
Thank you for reaching out to us and sharing your sentiments.
Hopefully, Frank is in a better place now and every time someone here thinks positively of him, things get better for him.

:)
Ashwin M

**Pull request merged**